src/view/configuration/configuration.jsx

```diff
@@ -521,17 +521,18 @@ const Configuration = ({ formikProps, initInfo }) => {
                       </div>
                     </div>
                     <div className="u-gapTop">
-                      <InfoTipLayout tip="Prehiding snipped should be placed on the page at the very top">
+                      <InfoTipLayout tip="Place the prehiding snippet within the <head> tag of the HTML page.">
                         <FieldLabel
-                          labelFor="prehidingStyleField"
+                          labelFor="copyToClipboardPrehidingStyleBtn"
```

Comment:
For data-test-ids, at least, we've been using the suffix "Button" instead of "Btn". I'd recommend being consistent with that.

Reply:
thank you